axlearn/experiments/text/gpt/fuji.py

```diff
@@ -86,6 +87,14 @@ class Version(enum.Enum):
 }
 
 
+# Regex patterns for matching remat names
+class RematRegex(enum.Enum):
+    QKV_PROJ = r".*\.?(k|q|v)_proj"
+    LINEAR1_X = r".*\.?linear1_[01]"
+    RESIDUAL_ADD = r"TransformerAttentionLayer\.residual_add"
+    MLP_RESIDUAL = r"TransformerFeedForwardLayer\.mlp_residual"
```

Comment:
How do we check whether these regex match the right activations?

Consider adding a test as in https://github.com/apple/axlearn/blob/a15a3bcbb976c14db157a8958df368a48c614c1f/axlearn/common/attention_test.py#L3831-L3876.

Reply:
Added a test